### PR TITLE
User templates on index.html

### PR DESCRIPTION
**Pull request by cmbrose**
This supports the auth refresh issue on the client by populating the user email template field in `index.html`.

### CODE REVIEW

**Comment by cmbrose** [comment]
https://github.com/lighthouse/lighthouse-client/pull/34

File: lighthouse.go [hunk 1]
```diff
@@ -34,7 +37,29 @@ const (
...
+    }
+
+    indexTemplate, err := template.New("index.html").ParseFiles(indexPath)
+    if err == nil {
+        err = indexTemplate.Execute(w, authData)
+    } 
+
+    if err != nil {
+        w.WriteHeader(404)
+    }
```
Comment (nickmonad) [inline lighthouse.go:62]: Should there be a call to `http.ServeFile` here? After restarting lighthouse and doing a refresh at the /beacons page, lighthouse just logs `GET /beacons` (which should be a 404) and waits.

File: auth/auth.go [hunk 2]
```diff
@@ -90,12 +90,16 @@ func AuthMiddleware(h http.Handler, ignorePaths []string) http.Handler {
...
 
-        if loggedIn := session.GetValueOrDefault(r, "auth", "logged_in", false).(bool); loggedIn {
+        if session.GetValueOrDefault(r, "auth", "logged_in", false).(bool) {
             h.ServeHTTP(w, r)
             return
         }
 
-        http.Redirect(w, r, "/login",  http.StatusFound)
+        if strings.Contains("/api", r.URL.Path) {
+            http.WriteHeader(401)
```
Comment (nickmonad) [inline auth/auth.go]: http --> w

Comment (cmbrose) [inline auth/auth.go]: @ngmiller fixed

**Pull request closed without merging**